spec/fixtures/db/migrate/create_index_concurrently_safe/20230309014016_create_index_concurrently.rb

```diff
@@ -11,10 +11,6 @@ def up
     execute <<~SQL
     CREATE INDEX CONCURRENTLY some_notifications_index ON notifications(user_id);
     SQL
-
-    execute <<~SQL
-    DROP INDEX some_notifications_index;
-    SQL
```

Comment:
Not necessary because the migration will never reach this step as index cannot be created concurrently in the `test` env since there is an active transaction.